docs/learn/learn-guides-bounties.md

```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
 
 ## Assign a Curator to a Bounty
 
+:::info
+
+The curator assignment call can only be executed successfully if the original bounty proposal
+funding has been executed and a new spending period has not begun. Proposers need to ensure they
+calculate a time further past the new spending period for the execution of their curator assignment
+call. Suppose your bounty has been awarded and gets funded at the end of the current spending
+period. In that case, you need to make sure that if the curator assignment ends before the end of
```

Comment:
"Curator assignment ends"  - do you mean "Curator assignment referenda" is approved before the end of the spend period and the enactment of the call needs to happen after?